.circleci/config.yml

```diff
@@ -1,9 +1,21 @@
-version: 2
-jobs:
-  checkout_and_test:
+version: 2.1
+
+executors:
+  mongo3_6:
     docker:
       - image: circleci/node:10
       - image: circleci/mongo:3.6-jessie
+  mongo4_4:
+    docker:
+      - image: circleci/node:10
```

Comment:
can we also update all the node versions to 14? here it's `circleci/node:10` and further down it's `circleci/node:8-stretch`

Reply:
Yes I can
What about running jobs for both of 10 and 14 versions , but use image built with mongo 4.4 and node 14 with for publishing?

Reply:
Sounds good, though I think if we're doing multiple Node builds, then we should do 10, 12 and 14?

Reply:
Yes we can. In this case there will be 6 jobs

Reply:
How **build_and_publish** workflow looks like now. I also added a conditional step **save_cache** for the job with **mongo:4.4** and  **node:14** docker images. From my view it doesn't make sense to cache images for each of the jobs.

<img width="525" alt="Screen Shot 2021-01-26 at 12 18 32" src="https://user-images.githubusercontent.com/15375081/105838703-7e872b00-5fe1-11eb-8bdb-6b8f3d356689.png">